src/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Leaderboard Project</title>
+</head>
+<body>
+    <h1 class="maintitleclass" id="maintitle">Leaderboard</h1>
+    <main>
+        <section class="left-section">
+          <header class="left-header">
+            <h2 class="left-heading">Recent Scores</h2>
+            <button class="button">Refresh</button>
+          </header>
+          <ul class="score-list">
+            <li class="item">Name : 100</li>
+            <li class="item">Name : 20</li>
+            <li class="item">Name : 35</li>
+            <li class="item">Name : 60</li>
+            <li class="item">Name : 89</li>
+            <li class="item">Name : 74</li>
+            <li class="item">Name : 48</li>
+          </ul>
```

Comment:
- [ ] Good job making your app look similar to the wireframe 👍 however according to the requirements HTML and CSS should be used just to make the layout, adding the data should be done using ES6 modules which include: the usage of arrow functions instead of the function keyword, usage of import and export instead of require or module.exports & proper usage of const and let keywords

Reply:
Fixed this problem. Used import/export, arrow function.